src/screens/FeedScreen/Content.js

```diff
@@ -25,7 +26,8 @@ const Content = ({
   topics = [],
   item,
   onNewPollFetched,
-  setHaveSeeMore
+  setHaveSeeMore,
+  hasComment
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new prop `hasComment` is introduced. Ensure that all instances where the `Content` component is used are updated to pass this prop if necessary.
```diff
+  hasComment
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->